zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/exporter/ExporterMetricsIT.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.exporter;
+
+import io.camunda.zeebe.qa.util.actuator.PrometheusActuator;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class ExporterMetricsIT {
+
+  @TestZeebe
+  private final TestStandaloneBroker zeebe =
+      new TestStandaloneBroker()
+          .withExporter(
+              "foo", cfg -> cfg.setClassName(ExporterMetricsTestExporter.class.getName()));
+
+  @Test
+  public void shouldAddMeterToExporterMetrics() throws IOException {
+    // given
+    final var actuator = PrometheusActuator.of(zeebe);
+    final var metricsPlainText =
+        IOUtils.toString(actuator.metrics().body().asInputStream(), StandardCharsets.UTF_8);
```

Comment:
:x: This isn't required anymore with newer Java versions. You can just do `new String(actuator.metrics().body().asInputStream().readAllBytes(), StandardCharsets.UTF_8);`. 

That said, Feign is pretty smart - did you try just returning `String` in the `PrometheusActuator` instead of `Response`? I have a feeling it might just do the whole thing for you.

I would try the following, assuming Feign can convert the body by itself to `String`:

```java
@Test
  public void shouldAddMeterToExporterMetrics() {
    // given
    final var actuator = PrometheusActuator.of(zeebe);

    // when
    final var metricsPlainText = actuator.metrics();

    // then
    Assertions.assertThat(metricsPlainText.lines())
        .filteredOn(line -> line.startsWith(ExporterMetricsTestExporter.REGISTERED_COUNTER))
        .hasSize(1);
  }
```